optax/_src/alias.py

```diff
@@ -109,3 +109,13 @@ def sgd(learning_rate: float,
       transform.trace(decay=momentum, nesterov=nesterov),
       transform.scale(-learning_rate),
   )
+
+
+def yogi(learning_rate: float,
+              b1: float = 0.9,
+              b2: float = 0.999,
+              eps: float = 1e-3) -> GradientTransformation:
```

Comment:
There is still a formatting issue actually,
b1, b2, eps should be indented in line with learning_rate